desc/objectives/_free_boundary.py

```diff
@@ -686,57 +713,36 @@ def build(self, use_jit=True, verbose=1):
 
         super().build(use_jit=use_jit, verbose=verbose)
 
-    def compute(self, params, constants=None):
+    def compute(self, params=None, constants=None):
         """Compute boundary force error.
 
         Parameters
         ----------
-        params : dict
-            Dictionary of equilibrium degrees of freedom, eg Equilibrium.params_dict
-        constants : dict
-            Dictionary of constant data, eg transforms, profiles etc. Defaults to
-            self.constants
+        params :
```

Comment:
Even though you set the default value to None, you should describe what that stands for. Also, you seem to be not using 'params', consider deleting it completely.
```suggestion
        params : dict
                some explanation about the usage of params
```